test/image/mocks/indicator_datacard3.json

```diff
@@ -0,0 +1,140 @@
+{
+  "data": [{
```

Comment:
Nice mock!

For reference, on master this mock looks like


![image](https://user-images.githubusercontent.com/6675409/60541380-7c62ca80-9cdf-11e9-8e12-359e9c48c607.png)

Reply:
It looks fine on `master` but if you change the font-size of delta or number, the alignment might break.